Ingress path issue in default generated Chart

The example Helm chart generated via `helm create` throws this error when the value for `ingress` is set to `true` and the rest of the values are left as the default:

```
[ERROR] templates/: template: test/templates/ingress.yaml:35:21: executing "test/templates/ingress.yaml" at <.path>: can't evaluate field path in type interface {}
```

This looks to be because the change introduced in #9057 didn't update the corresponding default section generated in `values.yaml`.

Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.2", GitCommit:"23dd3af5e19a02d4f4baa5b2f242645a1a3af629", GitTreeState:"dirty", GoVersion:"go1.15.5"}
```

Also tested with 3.5.0

Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.1", GitCommit:"c4d752765b3bbac2237bf87cf0b1c2e307844666", GitTreeState:"clean", BuildDate:"2020-12-19T08:38:20Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4+k3s2", GitCommit:"8f9e5628c1b5f0a673cde6051cea1afb65dcff7f", GitTreeState:"clean", BuildDate:"2020-12-07T21:30:05Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

Cloud Provider/Platform (AKS, GKE, Minikube etc.): 

Various (K3s, Kind)

--- a/pkg/chartutil/create.go
+++ b/pkg/chartutil/create.go
@@ -152,7 +152,11 @@ ingress:
     # kubernetes.io/tls-acme: "true"
   hosts:
     - host: chart-example.local
-      paths: []
+      paths:
+      - path: /
+        backend:
+          serviceName: chart-example.local
+          servicePort: 80
   tls: []
   #  - secretName: chart-example-tls
   #    hosts:

